src/Peering/Peering/Peering/NewAzurePeeringCommand.cs

```diff
@@ -132,6 +133,10 @@ public class NewAzurePeeringCommand : PeeringBaseCmdlet
             Mandatory = true,
             HelpMessage = Constants.PeeringExchangeConnectionHelp,
             ParameterSetName = Constants.Exchange)]
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = Constants.PeeringExchangeConnectionHelp,
+            ParameterSetName = Constants.ParameterSetNameConvertLegacyPeering)]
```

Comment:
@rileymckenna it looks like this `-ExchangeConnection` parameter is already a part of the `PeeringExchangeConnectionHelp` parameter set -- any reason this change is being made?

Reply:
I swapped the positioning of help message and parameterset, i fixed it so its consistent. Also we added exchange and direct connections to the legacy object because now that we have a connection identifier (2019-06-01 api) we can handle cases of adding new connections and not risk dropping the data.